oci/defaults.go

```diff
@@ -154,6 +154,10 @@ func DefaultLinuxSpec() specs.Spec {
 			{Type: "pid"},
 			{Type: "ipc"},
 		},
+		// Always use MS_PRIVATE for rootfs propagation to make sure that we
+		// don't leak any MS_UNBINDABLE mounts into the container's namespace
+		// while its resolving volume bindmounts.
+		RootfsPropagation: "private",
```

Comment:
> I thought somebody sets rootfsPropagation MS_PRIVATE | MS_REC in normal cases so libcontainer default does not kick in. Can't find though who does it.

It's the libcontainer default, but when generating the spec the *actual* default is `MS_PRIVATE|MS_REC` if you look at [the spec generation](https://github.com/opencontainers/runc/blob/v1.0.0-rc4/libcontainer/specconv/spec_linux.go#L40).